.github/workflows/roundup.yml

```diff
@@ -0,0 +1,26 @@
+name: weekly-summary
+
+on: 
+  schedule:
+   - cron: "15 19 * * FRI"
+
+jobs:
+  roundup-summary:
+    runs-on: ubuntu-latest
+    permissions:
+      issues: read
+    steps:
+      - name: checkout repo
+        uses: actions/checkout@v2
+      - name: setup python
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.10.1
+      - name: install deps
+        run: pip install requests
+      - name: execute script
+        run: |
+          python -u ./.github/actions/roundup_summary.py
+        env:
+          github_api_token: ${{ secrets.GITHUB_TOKEN }}
+          mailgun_api_key: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
wrong secret here

Reply:
Good catch, I updated the token to use the one from the [new-bugs-announce notifier](https://github.com/python/cpython/blob/42fee931d055a3ef8ed31abe44603b9b2856e04d/.github/workflows/new-bugs-announce-notifier.yml#L19) script.